modules/openy_gc_auth/modules/openy_gc_auth_daxko_sso/openy_gc_auth_daxko_sso.install

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * @file
+ * Install file for openy_gc_auth_personify module.
```

Comment:
Please fix the typo, this is `openy_gc_auth_daxko_sso` module

Reply:
@dchubar-fj, please use suggestions in this case. It is much faster to work with.